### PR TITLE
added some translations

### PR DESCRIPTION
**Pull request by memtech3**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered docs/spec/dark.md](https://github.com/memtech3/ant-design/blob/patch-3/docs/spec/dark.md)

### CODE REVIEW

**Comment by memtech3** [comment]
did I do this right?

File: docs/spec/dark.md [hunk 1]
```diff
@@ -5,12 +5,14 @@ category:
...
+  en-US: Dark Mode
 ---
 
-暗黑模式是指把所有 UI 换成黑色或者深色的一个主题模式。
+zh-CN: 暗黑模式是指把所有 UI 换成黑色或者深色的一个主题模式。
+en-US: Dark mode is a theme that makes all UI elements darker.
 
-## 何时使用
+zh-CN: ## 何时使用
+en-US: ## When to use
```
Comment (afc163) [inline docs/spec/dark.md]: 改错了。

Comment (memtech3) [reply docs/spec/dark.md]: can you explain the correction I can't see it
你能解释我看不到的更正吗

Comment (memtech3) [reply docs/spec/dark.md]: I am using google translate so excuse my grammar
我正在使用Google翻译，请原谅我的语法

Comment (afc163) [reply docs/spec/dark.md]: You can try the translation procedure described in https://github.com/ant-design/ant-design/issues/6931

Comment (memtech3) [reply docs/spec/dark.md]: ah thanks I will try that
啊，谢谢，我会尝试的

**Comment by memtech3** [comment]
I will close this pull request and restart to create a seperate file for english and chinese

**Pull request closed without merging**